aip-console-jenkins/README.md

```diff
@@ -341,6 +341,7 @@ Under `Advanced Settings` you will find the following parameters :
 * *Application Name*: Application on which to perform Deep Analysis.
 * *Snapshot Name*: Name of the snapshot. CAST Imaging Console will automatically assign one to the application if this
   field is empty.
+* *sleep duration* : Amount of seconds used to fetch the ongoing job status (defaulted to **15s**).
```

Comment:
I would replace "Amount" by "Number" because "amount" is related to a quantity of money

Reply:
done with the earlier remark